src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <!-- Shared testing infrastructure for running E2E tests using selenium -->
+  <Import Project="$(SharedSourceRoot)E2ETesting\E2ETesting.props" />
```

Comment:
This import and the targets import below is all that's needed to bring in Selenium E2E tests